src/components/SidebarTabs/SharingSidebarTab.vue

```diff
@@ -390,6 +408,10 @@ export default {
 			delete newAccess.legacyLink
 			this.$emit('update:formProp', 'access', newAccess)
 		},
+
+		async openQrDialog(qrText) {
+			this.qrDialog = qrText
+		},
```

Comment:
```suggestion
		openQrDialog(share) {
			this.qrDialogText = getPublicShareLink(share)
		},
```

see above (also this is not async)